### ISSUE TITLE
Exporting a Workflow Doesnt work

### ISSUE THREAD

**Issue by markbond1007** [salesagility/SuiteCRM-Core#409]
#### Issue

Going to the Workflows list section (Admin/Workflow Manager), selecting 1 or more workflows and then Bulk Action/Export does nothing.

#### Expected Behavior

Should download a CSV export of the selected Workflows.

#### Actual Behavior

Nothing happens, cant see anything in the logs

#### Steps to Reproduce

1. Go to workflows
2.Select a workflow
3.Click Export
4.

#### Your Environment

* SuiteCRM Version used: Suite 8.4.1
* Browser name and version (e.g. Chrome Version 51.0.2704.63 (64-bit)): Chrome/Edge
* Environment name and version (e.g. MySQL, PHP 7): MariaDB, PHP8.0/PHP8.2
* Operating System and version (e.g Ubuntu 16.04): Debian/Ubuntu

**Comment by pgorod** [comment]
I don't think this ever worked. It's probably a case for just removing the Export option from the menu...

Workflows use more than one table, actions are separate, and the repeated runs info is another separate thing. They're not that simple to export, I'm afraid.

**Comment by markbond1007** [comment]
Ahh, well it works under Suite7 although looking back at whats exported you may be correct in that it doesnt export enough.

Thanks

Mark
